services/registration.js

```diff
@@ -83,8 +83,8 @@ router.get("/logout", auth, async (req, res) => {
     if (!user) {
       return res.status(401).json({ message: "Not authorized" });
     }
```

Comment:
to nie jest potrzebne możesz usunąć tym zajmuje się middelwere auth
```
    const user = await User.findById(req.user._id);
    if (!user) {
      return res.status(401).json({ message: "Not authorized" });
    }
 ```
 i to samo w /current